acj/static/modules/assignment/assignment-module_spec.js

```diff
@@ -47,7 +47,12 @@ describe('course-module', function () {
 		"description": null,
 		"id": 1,
 		"modified": "Fri, 09 Jan 2015 17:23:59 -0000",
-		"name": "Test Course"
+		"name": "Test Course",
+        "year": 2015,
```

Comment:
Thanks. I'm actually in favor of spaces as well.

On Thu, Jul 14, 2016 at 1:33 PM Andrew Gardener <notifications@github.com>
wrote:

> In acj/static/modules/assignment/assignment-module_spec.js
> <https://github.com/ubc/acj-versus/pull/313#discussion_r70878350>:
>
> > @@ -47,7 +47,12 @@ describe('course-module', function () {
> >  		"description": null,
> >  		"id": 1,
> >  		"modified": "Fri, 09 Jan 2015 17:23:59 -0000",
> > -		"name": "Test Course"
> > +		"name": "Test Course",
> > +        "year": 2015,
>
> eaf78d5
> <https://github.com/ubc/acj-versus/commit/eaf78d522cb895b08bfec89d26c87625742bb219>
> I converted all tab intention for .html and .js files to 4 spaces to match
> the indention for the .py files
>
> I realize the dev doc says that the project uses tabs for indention but
> none of the python files were following it so I opted to go in favor of
> spaces (the html and js files were mixed indention)
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/ubc/acj-versus/pull/313/files/5aadfd51ab1b4143ef955a55c80e858b9fc57087#r70878350>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAcajG_KnxlUCETbjn6yNSYqngsSOvhDks5qVp0KgaJpZM4JK2Ht>
> .
>